content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst

```diff
@@ -22,40 +22,49 @@ Product
 -------
 
 To auto-populate product information and prices on an |RFQ|, configure products by going to
-:menuselection:`Purchase app --> Products --> Products`. Select an existing product, or create a
-new one by selecting :guilabel:`New`. Doing so opens the product form, where various sales and
-purchasing data can be configured.
+:menuselection:`Purchase app --> Products --> Products`. Select an existing product, or create a new
+one by selecting :guilabel:`New`. Doing so opens the product form, where sales and purchasing data
+can be configured.
 
-To configure purchasable products, tick the :guilabel:`Can be purchased` checkbox, under the product
-name. Then, go to the :guilabel:`Inventory` tab, and enable the :guilabel:`Buy` route.
+To configure purchasable products, tick the :guilabel:`Purchase` checkbox, under the product name.
+Next, go to the :guilabel:`Inventory` tab, and enable the :guilabel:`Buy` route.
+
+.. important::
+   The :guilabel:`Inventory` tab and routes are only visible if using the :doc:`Inventory app
+   <../../inventory>`.
+
+.. seealso::
+   :doc:`Configure product types and track quantities <../../../inventory_and_mrp/inventory/product_management/configure>`
 
 .. image:: rfq/product-vendor-pricelist-config.png
-   :align: center
    :alt: Required configuration for purchasable products.
 
+
 .. _purchase/manage_deals/vendor-pricelist:
 
 Vendor pricelist
 ----------------
 
-In the :guilabel:`Purchase` tab of the product form, input the vendor and their price, to have this
-information auto-populate on an |RFQ| each time the product is listed.
+In the :guilabel:`Purchase` tab of the product form, click :guilabel:`Add a line` to input the
+vendor and their price, to have this information auto-populate on an |RFQ| each time the product is
+listed.
 
 .. seealso::
    :doc:`../products/pricelist`
 
-Default columns include :guilabel:`Quantity`, :guilabel:`Price`, and :guilabel:`Delivery Lead Time`,
-but other columns like, :guilabel:`Product Variant` or :guilabel:`Discounts`, can also be enabled.
+Default columns include :guilabel:`Quantity`, :guilabel:`Unit Price`, and :guilabel:`Delivery Lead
+Time`, but other columns like, :guilabel:`Vendor Product Code` or :guilabel:`Discount (%)`, can also
+be enabled.
 
 To enable or disable columns, click the :icon:`oi-settings-adjust` :guilabel:`(additional options)`
 icon on the right side of the header row to reveal a drop-down menu of additional columns that can
 be added (or removed) from the :guilabel:`Purchase` tab.
```

Comment:
Essentials content, can remove
It's better to more deeply explain the what default columns mean and how to fill them in correctly.
```suggestion
```